src/main/java/com/strandls/naksha/layers/scripts/generate_geoserver_styles.java

```diff
@@ -217,8 +212,9 @@ private static void generate_style(List<String> tables, String geoserver_data_di
 		try {
 			String dbhost = NakshaConfig.getString("geoserver.dbhost");
 			String dbport = NakshaConfig.getString("geoserver.dbport");
-			connection = DriverManager.getConnection("jdbc:postgresql://" + dbhost + ":" + dbport + "/" + db, user, pass);// +db, user
-																											// ,pass
+			connection = DriverManager.getConnection("jdbc:postgresql://" + dbhost + ":" + dbport + "/" + db, user,
+					pass);// +db, user
```

Comment:
don't add ambiguous comments
this won't help anyone else understand what is it about either add descriptive comment or remove it